### ISSUE TITLE
Failure to provide Reliability analysis

### ISSUE THREAD

**Issue by AmandioDias** [arcaldwell49/SimplyAgree#49]
Hello,
When i try to run a reliability analysis and add columns to the measurements, ir gives a error: "no vector columns were selected"
I have run it several times and always comes back with the same error. Please see the attached image.

Could you please help me to see what am i doing wrong?

Thank you

![screenshot](https://github.com/arcaldwell49/SimplyAgree/assets/157042567/c9d920f7-e4ad-4c6e-bc22-2d9b9b4fd14d)

**Comment by arcaldwell49** [comment]
I'd need a reproducible example to help. Everything is working on my end with a new download of jamovi. My guess is there is an error being introduced by the data itself

**Comment by AmandioDias** [comment]
Hello,
Thank you for the reply.
I have attached a CSV file with the data. Hope it helps.

[Dados juntos.csv](https://github.com/arcaldwell49/SimplyAgree/files/13980312/Dados.juntos.csv)

**Comment by arcaldwell49** [comment]
Yeah, still not reproducible. My best guess is who have some data management issues on your end. [Try renaming the columns](https://b-greve.gitbook.io/beginners-guide-to-clean-data/database-related-problems/naming-tables-and-columns) and checking the data. I did notice you have an unequal number of columns in the dataset. 

TL;DR, I have no proof this is a SimplyAgree related problem.
